app/models/card.rb

```diff
@@ -34,17 +35,30 @@ def update_from_pack(user, card_params)
   end
 
   def verify_translation(user_translation)
-    if transform_string(original_text) == transform_string(user_translation)
+    typos = Text::Levenshtein.distance(transform_string(original_text), transform_string(user_translation))
+    if typos < 3
+      processing_success_translation_update
+      { result: true, typos: typos }
+    else
+      processing_failure_translation_update
+      { result: false, typos: typos }
+    end
+  end
+
+  def processing_success_translation_update
```

Comment:
Assignment Branch Condition size for processing_success_translation_update is too high. [16.12/15]